cachito/workers/tasks/npm.py

```diff
@@ -32,6 +32,51 @@
 log = logging.getLogger(__name__)
 
 
+def _verify_npm_files(bundle_dir, subpaths):
+    """
+    Verify that the correct are present for the npm package manager to proceed.
+
+    :param RequestBundleDir bundle_dir: the ``RequestBundleDir`` object for the request
+    :param list subpaths: a list of subpaths in the source repository of npm packages
+    :raises CachitoError: if the repository is missing the required files or contains invalid
+        files/directories
+    """
+    for subpath in subpaths:
+        bundle_dir_subpath = bundle_dir.app_subpath(subpath)
+        for lock_file in (
+            bundle_dir_subpath.npm_shrinkwrap_file,
+            bundle_dir_subpath.npm_package_lock_file,
+        ):
+            if lock_file.exists():
+                break
+        else:
+            shrinkwrap_rel_path = bundle_dir_subpath.get_relpath(
```

Comment:
It would be great if we didn't have to duplicate these entries here. Could we do something like:
```python
lock_files = (bundle_dir_subpath.npm_shrinkwrap_file, bundle_dir_subpath.npm_package_lock_file)

...

lock_files_relpath = [bundle_dir_subpath.get_relpath(lock_file) for lock_file in lock_files]
```